pkg/mcs/tso/server/server.go

```diff
@@ -574,7 +574,7 @@ func (s *Server) startServer() (err error) {
 
 	s.participant = member.NewParticipant(s.etcdClient)
 	s.participant.InitInfo(uniqueName, uniqueID, fmt.Sprintf(tsoSvcDiscoveryPrefixFormat, s.clusterID, mcsutils.DefaultKeyspaceID),
```

Comment:
Do we need to add AdvertiseListenAddr to tsopb.Participant?

Reply:
It seems that tsopb.Participant just need to store AdvertiseListenAddr as ListenAddr for this case?

Reply:
Yes. There is no need to change the definition of `Participant`.